airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/contact_lists.json

```diff
@@ -54,7 +54,7 @@
               "type": ["null", "string"]
             },
             "value": {
-              "type": ["null", "string"]
+              "type": ["null", "string", "integer", "boolean"]
```

Comment:
Can we shift type validation process to `get_json_schema`? And test it by unit tests.